lib/SystemManagement/Ghosts.pm

```diff
@@ -40,10 +40,11 @@ sub CheckGhostFor {
 }
 
 sub ParseGhosts {
-	my($depth,@what) = @_;
+	my ($depth, @what) = @_;
 
-	if ($depth>10) {
-		print STDERR "ParseGhosts: recurrsive macro?  Depth == 10 at '".join(',',@what)."'\n";
```

Comment:
I think this was originally to avoid the backtrace, but I don't see a good reason to avoid it.